nuxt-app/pages/index.vue

```diff
@@ -1,6 +1,105 @@
 <template>
-    <div>
-        <h1>Nuxt - PIE Aperture</h1>
-        <NuxtLink to="/form">Form page</NuxtLink>
-    </div>
-</template>
\ No newline at end of file
+      <h1>Nuxt - PIE Aperture</h1>
+      <NuxtLink to="/form">Form page</NuxtLink>
+      <h2>PIE Button</h2>
+      <pie-button @click="setCounter">
+        Counter: {{ counter }}
+      </pie-button>
+      <pie-divider/>
+      <h2>PIE Icon Button</h2>
+      <pie-icon-button>
+        <icon-close/>
+      </pie-icon-button>
+      <pie-divider/>
```

Comment:
Very nit-picky but can we get some whitespace up in this file, like on either side of the dividers, etc?